gateway/gateway.go

```diff
@@ -510,7 +517,11 @@ func (gateway *HandleT) pixelHandler(w http.ResponseWriter, r *http.Request, req
 			req.Header.Add("X-Forwarded-For", r.Header.Get("X-Forwarded-For"))
 
 			// get web payload
-			gateway.setWebPayload(req, queryParams, reqType)
+			err := gateway.setWebPayload(req, queryParams, reqType)
```

Comment:
Update the above comment with more details like: 
This converts pixel request to web request.